common/app/model/content/Atoms.scala

```diff
@@ -209,6 +211,10 @@ final case class TimelineItem(
 
 object Atoms extends common.Logging {
 
+  def articleConfig = ArticleConfiguration(
```

Comment:
Should this be moved in `Configuration` as a `toAtomArticleConfiguration` or something?